web/src/main/java/org/phoenixctms/ctsms/web/model/IDVO.java

```diff
@@ -16,16 +16,8 @@ public static IDVO transformVo(Object object) {
 		return new IDVO(object);
 	}
 
-	public static void transformVoCollection(Collection<?> vos) {
-		if (vos != null) {
-			CollectionUtils.transform(vos, new Transformer() {
-
-				@Override
-				public Object transform(Object input) {
-					return transformVo(input);
-				}
-			});
-		}
+	public static List<IDVO> transformVoCollection(Collection<?> vos) {
+		return vos.stream().map(IDVO::new).collect(Collectors.toList());
```

Comment:
think the the original idea of the collection "transform" was to safe (do it in) memory.

Reply:
has been reverted